### DISCUSSION TITLE
希望可以增加一个自定义搜索词的功能

### DISCUSSION THREAD

**Discussion by AkimioJR** [jxxghp/MoviePilot#3467] [讨论]
### 当前程序版本

v.1.7.7

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望可以和 nastools 一样使用自己的搜索词对站点资源进行检索，一些 web资源 动漫往往采用 台湾巴哈姆特的译名或者是制作组自己的译名，和 tmdb 上的正式译名可能略有差异导致检索不到种子
我看目前的检索方式是中文名+英文名+原名，希望可以再加一个用户搜索时的名字，感谢大大

**Comment by omo00omo** [comment]
订阅编辑中修改 搜索关键词，可以订阅MP中检索不到的种子。就是麻烦点，需要在站点提前测试检索关键词。
若可以同时在多个站点搜索关键词，确实会方便不少。
